From: David Cohen <david.a.cohen@intel.com>
Subject: usb: gadget: android: fix compilation warning

This patch fixes the following compilation warning:
drivers/usb/gadget/android.c: In function 'functionfs_ready_callback':
drivers/usb/gadget/android.c:424:1: warning: label 'err' defined but not used [-Wunused-label]

Signed-off-by: David Cohen <david.a.cohen@intel.com>
---
 drivers/usb/gadget/android.c |    1 -
 1 file changed, 1 deletion(-)

--- linux.orig/drivers/usb/gadget/android.c
+++ linux/drivers/usb/gadget/android.c
@@ -421,7 +421,6 @@ static int functionfs_ready_callback(str
 	if (config->enabled)
 		android_enable(dev);
 
-err:
 	mutex_unlock(&dev->mutex);
 	return ret;
 }
